README.md

```diff
@@ -56,17 +61,17 @@ After that call the install.sh script.
 
 The following script should do everything for you:
 ```
-wget https://github.com/vikt0rm/dbus-shelly-1pm-pvinverter/archive/refs/heads/main.zip
-unzip main.zip "dbus-shelly-1pm-pvinverter-main/*" -d /data
-mv /data/dbus-shelly-1pm-pvinverter-main /data/dbus-shelly-1pm-pvinverter
-chmod a+x /data/dbus-shelly-1pm-pvinverter/install.sh
-/data/dbus-shelly-1pm-pvinverter/install.sh
+wget https://github.com/vincegod/dbus-shelly-em-smartmeter/archive/refs/heads/main.zip
+unzip main.zip "dbus-shelly-em-smartmeter-main/*" -d /data
+mv /data/dbus-shelly-em-smartmeter-main /data/dbus-shelly-em-smartmeter
+chmod a+x /data/dbus-shelly-em-smartmeter/install.sh
+/data/dbus-shelly-em-smartmeter/install.sh
```

Comment:
@ramack With this change code of this codebase is not used anymore. Why this PR then? Either we make one repo and one solution for metering 1 phase systems and using this as a pv inverter, or we decide on having two separate solution. I see you already have met the decision.